ui/app/templates/workspace/projects/new.hbs

```diff
@@ -0,0 +1,51 @@
+<AppBreadcrumbs/>
+<PageHeader @iconName="plus-plain">
+  <div class="title">
+    <h1>New Project</h1>
```

Comment:
I asked this before, but can't remember what your reply was, so I'll ask it again (sorry! 😬 ) - are y'all i18n'ing things?

Reply:
no worries, I need reminders! For context, I started working on this branch in parallel with the previous PR's, so I hadn't followed your suggestions yet 👍